daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/DarReaderTest.scala

```diff
@@ -49,6 +49,7 @@ class DarReaderTest extends WordSpec with Matchers with Inside with BazelRunfile
               ((packageId15, archive15), LanguageMajorVersion.V1) ::
               ((packageId16, archive16), LanguageMajorVersion.V1) ::
               ((packageId17, archive17), LanguageMajorVersion.V1) ::
+              ((packageId18, archive18), LanguageMajorVersion.V1) ::
```

Comment:
Is there a way to make this less repetitive?

Reply:
We definitely can and I would go as far as saying that all tests that break just because there is a new stable package are broken but I don’t want to fix this in this PR.